### DISCUSSION TITLE
民眾有感的永續科技，培養跨界視野的科學人才

### DISCUSSION THREAD

**Discussion by Guagua1234** [1101-GS4719-C/Python-Community-GS4719-C#1132] [General]
我認為吳政忠對於如何實現科技政策的永續目標的看法是他認為科技效益要讓大眾從日常生活中體察到，國科會的著力點有兩大方向，一是基於前瞻基礎建設計畫，建構民生公共物聯網，針對水、空、地、災議題，找出可行的科技解決方案，二是透過各種科普推廣計畫，吸引更多新世代人才投入科研，這兩個我都覺得對社會很有幫助。
https://www.thenewslens.com/feature/digi-plus/174003

**Comment by woohsin** [comment]
透過科普推廣計畫吸引更多新世代人才投入科研，也是非常重要的。新世代人才的加入可以注入新的活力和創意，推動科技領域的不斷發展，同時也有助於培養未來的科技領袖和專家，為國家的科技創新打下良好的基礎。

總的來說，吳政忠的觀點確實對於實現科技政策的永續目標具有重要意義，尤其是在讓科技成果更貼近民生、更具有可見度，以及吸引新世代人才投入科研等方面。

**Comment by titan92601** [comment]
吳政忠的科技政策觀點強調了科技對大眾生活的實質影響，並提出具體措施以解決社會問題和吸引更多人參與科研。我認為這些方向有助於促進科技與社會的融合，提升科技的公共效益。此外，透過科普推廣，可以提高公眾對科技的認知，培養新世代的科技人才，推動科技創新和可持續發展。

**Comment by NcuJerryKao** [comment]
吳政忠對於實現科技政策永續目標的看法提出了非常具有建設性的方向，他強調科技效益應讓大眾在日常生活中切實感受到，這樣才能真正推動社會進步。國科會的著力點主要有兩個方向，首先是基於前瞻基礎建設計畫，建構民生公共物聯網，這對於解決現代社會中與水、空氣、土地及災害相關的問題提供了科技解決方案。這種應用型科技的推廣，不僅能夠提升民眾的生活品質，還能有效地應對各種環境和災害挑戰，提高社會的整體韌性和可持續發展能力。

此外，吳政忠強調透過各種科普推廣計畫，吸引更多新世代人才投入科研，這同樣是非常重要的。當前社會的快速發展和變化，需要更多具有創新思維和專業技能的年輕人才。科普推廣計畫不僅能激發年輕人對科學的興趣和熱情，還能為未來的科技創新和發展奠定堅實的基礎。這些計畫可以包括學校教育、科學節日、科技博覽會等多種形式，讓更多人了解和參與到科技中來，形成良性循環，推動整個社會的科技進步。

總的來說，科技政策的永續發展需要將科技成果轉化為民眾可感知的實際效益，同時不斷培養和吸引新的人才。這兩者相輔相成，缺一不可。通過這些措施，不僅能夠提高社會的科技水平和創新能力，還能實現科技與社會的良性互動，推動社會的可持續發展。希望未來能看到更多這樣具體而有力的政策落實，讓科技真正成為推動社會進步的重要力量。

**Comment by LinChloe** [comment]
我也認為民眾對永續科技的感受激發了對跨界視野的需求，培養科學人才至關重要。透過跨領域合作，科學家可以結合不同專業知識，創新解決永續挑戰。這種交流不僅促進了科技進步，也提升了專業人士的意識和技能，讓他們更有能力應對現實挑戰。世界越來越美好~

**Comment by hbh10601** [comment]
永續科技是一項全面的社會轉型，目的在於建立一個更加和諧、平衡的生態與經濟體系，除了注重資源利用和環境保護外，它還著眼於社會公平和人類福祉。
這其中也包括需要提高技術普及率，確保所有人都能夠分享科技進步所帶來的利益。
另外，永續科技也需要考慮到文化和歷史的因素，尊重和保護傳統知識和生態智慧。
我覺得這篇文章提供的方向很好！

**Comment by alexender0104** [comment]
我認為要創新，就必須有系統化改革，例如過去我們都避免犯錯，這與創新是格格不入的，而政府組織如果只仰賴單一部會，缺乏整合是無法用國家層級進行科技轉型。培養跨界視野的科學人才對於推動永續科技的發展至關重要。這樣的人才需要具備不同領域的知識，能夠跨越學科界限，將不同領域的專業知識和技術結合起來，提出創新的解決方案。他們不僅需要具備扎實的科學基礎知識，還需要具備創意思維和解決問題的能力，以應對日益複雜和跨學科的挑戰。

**Comment by Shang621** [comment]
民眾感知的永續科技包括太陽能、風能和生物能等，這些技術在解決能源和環境問題上具有顯著影響。培養跨界視野的科學人才至關重要，他們能夠跨越不同領域，將科學知識應用於解決複雜的永續發展問題。這需要結合不同學科的知識，培養創新思維和解決問題的能力，以應對日益複雜的挑戰。
